src/Annotator/ExtraPropertyAnnotator.php

```diff
@@ -269,14 +269,14 @@ private function makePageIdDataItem() {
 		}
 	}
 
-	private function makePageLengthItem() {
-		$pageLen = $this->getWikiPage()->getLen();
-		
+	private function makePageLengthDataItem() {
+		$pageLen = $this->getWikiPage()->getTitle()->getLength();
+
```

Comment:
Well do not really know why this is different to e.g. getLatest but I trust you on this one. :)